src/Installers/Windows/SharedFrameworkBundle/thm.wxl

```diff
@@ -7,7 +7,7 @@
   <String Id="ConfirmCancelMessage">Are you sure you want to cancel?</String>
   <String Id="ExecuteUpgradeRelatedBundleMessage">Previous version</String>
   <String Id="HelpHeader">Setup Help</String>
-  <String Id="HelpText">/install | /repair | /uninstall | /layout [\[]directory[\]] - installs, repairs, uninstalls or
+  <String Id="HelpText"><!--_locComment_text="'directory' is to be translated in both places"-->/install | /repair | /uninstall | /layout [\[]directory[\]] - installs, repairs, uninstalls or
```

Comment:
See 1b19eaf0d924aac9c21c0a808b376b9dbcf47076, where this string was broken in a few translated files and (unfortunately) checked into the repo.